### PR TITLE
Add todo-mvc-dijit

### PR DESCRIPTION
**Pull request by kitsonk**
**Enhancement**

This PR adds an example of using Dojo 1 Dijits integrated into a Dojo 2 application.

In particular, the `TodoItem` widget uses Dojo 1 Dijits of `CheckBox`, `TextBox`, and `Button` to provide some of the UI.

### CODE REVIEW

File: todo-mvc-dijit/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+- [Pre-requisites](#pre-requisites)
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
```
Comment (dylans) [inline todo-mvc-dijit/README.md]: Pre-requisites -> Prerequisites

File: todo-mvc-dijit/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+To use `todo-mvc`, clone the `examples` repository and install the npm dependencies:
+
+```shell
+git clone https://github.com/dojo/examples
+cd examples/todo-mvc-dijit
+npm install
+```
+
+To run the application simply run the dojo build.
```
Comment (dylans) [inline todo-mvc-dijit/README.md]: build. ->  build:

File: todo-mvc-dijit/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+```
+dojo build
+```
+
+To watch and serve the application source, then run the build with the `-w` command.
```
Comment (dylans) [inline todo-mvc-dijit/README.md]: command. -> command:

File: todo-mvc-dijit/package.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "dependencies": {
+    "@dojo/core": "beta2",
+    "@dojo/has": "beta2",
+    "@dojo/i18n": "beta2",
+    "@dojo/interop": ">=2.0.0-alpha.2",
+    "@dojo/routing": "beta2",
+    "@dojo/shim": "beta2",
+    "@dojo/widget-core": "beta2",
+    "dijit": "^1.12.2",
```
Comment (dylans) [inline todo-mvc-dijit/package.json]: 1.13.0

File: todo-mvc-dijit/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    "dojo": "^1.12.2",
```
Comment (dylans) [inline todo-mvc-dijit/package.json]: 1.13.0

File: todo-mvc-dijit/package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    "dojo-themes": "^1.12.2",
```
Comment (dylans) [inline todo-mvc-dijit/package.json]: 1.13.0

File: todo-mvc-dijit/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    "dojo-typings": "^1.11.8"
```
Comment (dylans) [inline todo-mvc-dijit/package.json]: Do we need to do another updated release for typings?

File: todo-mvc-dijit/src/main.ts [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+			filter: 'all'
+		},
+		defaultRoute: true
+	}
+];
+
+const router = registerRouterInjector(config, registry);
+const Projector = ProjectorMixin(TodoAppContainer);
+const projector = new Projector();
+// bug in widget-core that doesn't correctly promote registry
```
Comment (dylans) [inline todo-mvc-dijit/src/main.ts]: please add link to bug and cross-reference this in that bug so we remember to update this.

**Comment by agubler** [comment]
@kitsonk What's the plan with this PR?

**Comment by kitsonk** [comment]
Refresh it and try to get it landed for Beta 5

**Comment by agubler** [comment]
I think we should close this PR as `cli-build-app` currently doesn't support externals (it is a planned enhancement) and we probably shouldn't have any examples that still need to use the deprecated `cli-build-webpack` command.

**Pull request closed without merging**